codec-http2/src/main/java/io/netty/handler/codec/http2/Http2Stream.java

```diff
@@ -36,6 +36,14 @@
     }
 
     /**
+     * A key used for accessing user properties attached to the stream.
+     *
+     * @param <T> the property type to be stored.
+     */
+    interface Key<T> {
```

Comment:
There's really no need.  The details of the key are entirely hidden to the caller.